client/test/cypress/integration/account.spec.js

```diff
@@ -0,0 +1,43 @@
+/// <reference types="Cypress" />
+/// <reference path="../support/index.d.ts" />
+
+describe('Account', () => {
+  beforeEach(() => {
+    cy.task('resetDb');
+    cy.login({ email: 'regularuser@ccrproject.dev' });
+    cy.visit('/account/profile');
+  });
+
+  it('can visit the account profile page from the dropdown', () => {
+    cy.visit('/');
+    cy.dataCy('dropdown_username').click();
+    cy.dataCy('link_my_account').click();
+    cy.url().should('include', '/account/profile');
+  });
+
```

Comment:
This should move to header.spec.  I'll update that
```suggestion
```

Reply:
Sounds good

Reply:
If you didn't see it, I pushed a commit last night taking care of this.